src/scripting/lua_map.cc

```diff
@@ -7194,12 +7279,51 @@ int LuaShip::build_colonization_port(lua_State* L) {
    .. method:: make_expedition([items])
 
       Turns this ship into an expedition ship without a base port. Creates all necessary
-      wares and a builder plus, if desired, the specified additional items.
-      Any items previously present in the ship will be deleted.
-
+      wares and a builder plus, if desired, the specified additional **items**.
       The ship must be empty and not an expedition ship when this method is called.
 
-      :returns: nil
+      Note that the ships :attr:`capacity` is not adjusted if you give additional **itmes**.
```

Comment:
itmes → items

Reply:
I read it over and over but didn't saw this misspelling :-D